Command parasing with api

Koss24 opened the pull request:
Merged together command line parsing and yahoo api

dheadrick1618 reviewed (changes requested):
This is spitting out 'Error: FetchFailed ' when I give it a bad stock ticker. I wonder if we should instead be prompting the user to enter a new ticker and keep the program in a loop unless they exit? The project outline says the program cant' crash when given a bad stock symobl, so although its not technically crashing I think this give the appearance that the program isnt working properly .

dheadrick1618 commented on gitignore.txt:Zone.Identifier:
What is this for?

Koss24 commented:
Added a new commit, just removed the error println and handling the empty quotes in main.rs to inform the user of a potentially wrong or non-existent ticker
